setup.py

```diff
@@ -61,7 +61,7 @@
     "marshmallow>2.13.0,<=2.20.1",
     "reana-commons[kubernetes]>=0.9.3a1,<0.10.0",
     "reana-db>=0.9.1,<0.10.0",
-    "htcondor==8.9.11",
```

Comment:
This will need to bump HTCondor also in `Dockerfile`  in places like:

```
      wget -qO - https://research.cs.wisc.edu/htcondor/ubuntu/HTCondor-Release.gpg.key | apt-key add -; \
      echo "deb http://research.cs.wisc.edu/htcondor/ubuntu/8.9/focal focal contrib" >> /etc/apt/sources.list; \
      echo "deb-src http://research.cs.wisc.edu/htcondor/ubuntu/8.9/focal focal contrib" >> /etc/apt/sources.list; \
```

so that we have compatible version installed system-wide.

Note that we may also need to bump associated CERN files such as:

```
      wget -O ngbauth-submit.rpm https://linuxsoft.cern.ch/internal/repos/batch8s-stable/x86_64/os/Packages/ngbauth-submit-0.26-2.el8s.noarch.rpm; \
      wget -O myschedd.rpm https://linuxsoft.cern.ch/internal/repos/batch8s-stable/x86_64/os/Packages/myschedd-1.9-2.el8s.x86_64.rpm; \
```

to have corresponding versions. We can get there whilst testing.

Reply:
As discussed IRL, I'll revive my old tests and push onto your branch.